setup.py

```diff
@@ -197,7 +209,18 @@ def run(self):
     setup_args['entry_points'] = {
         'console_scripts': [
             'jupyter-nbconvert = nbconvert.nbconvertapp:main',
-        ]
+        ],
+        "nbconvert.exporters" : [
+            'custom=nbconvert.exporters:TemplateExporter',
```

Comment:
Yes and no. The custom exporter is configurable in regular config-files. So you can just use `--to custom` and it will pick up the rest of its configuration. It should _mostly_ be replaced by fully qualified exporter names, and entry points now.